### PR TITLE
back-end workspace initiated

### PR DESCRIPTION
**Pull request by Aloxbro**
back-end workspace created with dependencies "express", "nodemon" , also added index.js

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+front
```
Comment (rocambille) [inline .gitignore]: why ignoring the front directory?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Aloxbro) [reply .gitignore]: its only for the PR, to push back-end dir to dev instead of pushing front again

Comment (rocambille) [reply .gitignore]: to not push front again, just don't push it: don't add it, don't commit it, don't push it

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by Aloxbro** [comment]
Fixed

**Pull request merged**